lib/bloc/trending_movies_bloc.dart

```diff
@@ -20,6 +20,22 @@ class TrendingMoviesBloc extends ITrendingMovies {
     );
   }
 
+  void searchMovies(String query) async {
+    switch (query) {
+      case "":
```

Comment:
the "" can be a constant called emptyString

Reply:
Added.